README.md

```diff
@@ -1,138 +1,187 @@
+| ✅ | ✅ | ✅ | ✅ | ✅ | Coming... | Coming... |
 
-## Meetups
+Simple patterns for finding code in many languages:
 
-Want to learn more about sgrep? Check out these [slides from the r2c February meetup](https://r2c.dev/sgrep-public2.pdf)
+| **Pattern** | **Matches** |
+| :--- | :--- |
+| `$X == $X` | `if (node.id == node.id): ...` |
+| `requests.get(..., verify=False, ...)` | `requests.get(url, timeout=3, verify=False)` |
+| `os.system(...)` | `from os import system; system('echo sgrep')` |
+| `$ELEMENT.innerHTML` | ``el.innerHTML = "<img src='x' onerror='alert(`XSS`)'>";`` |
+| `$TOKEN.SignedString([]byte("..."))` | `ss, err := token.SignedString([]byte("HARDCODED KEY"))` |
 
 ## Installation
 
-Too lazy to install? Try out [sgrep.live](https://sgrep.live)
-
-### Docker
-
-`sgrep` is packaged within a [docker container](https://hub.docker.com/r/returntocorp/sgrep), making installation as easy as [installing docker](https://docs.docker.com/install/).
+For the best experience, simply install `sgrep` with [Docker](https://docs.docker.com/install/):
 
-## Quickstart
-
-```bash
-docker pull returntocorp/sgrep
+```
+$ docker pull returntocorp/sgrep
+```
 
-cd /path/to/repo
-# generate a template config file
-docker run --rm -v "${PWD}:/home/repo" returntocorp/sgrep --generate-config
+`sgrep` binaries can also be installed on Ubuntu:
 
-# look for findings
-docker run --rm -v "${PWD}:/home/repo" returntocorp/sgrep
+```
+$ wget https://github.com/returntocorp/sgrep/releases/download/v0.4.9/sgrep-0.4.9-ubuntu-16.04.tgz
+$ wget https://github.com/returntocorp/sgrep/releases/download/v0.4.9/sgrep-0.4.9-ubuntu-16.04.tgz.sha256
+$ sha256sum --check sgrep-0.4.9-ubuntu-16.04.tgz.sha256
+$ tar xf sgrep-0.4.9-ubuntu-16.04.tgz
+$ ./sgrep-lint-files/sgrep-lint --help
 ```
 
 ## Usage
 
-### Rule Development
+Start with a simple example:
 
-To rapidly iterate on a single pattern, you can test on a single file or folder. For example,
-
-```bash
-docker run --rm -v "${PWD}:/home/repo" returntocorp/sgrep -l python -e '$X == $X' path/to/file.py
+```
+$ cat << EOF > test.py
+a = 1
+b = 2
+if a == a:  # oops, supposed to be a == b
+    print('sgrep test')
+EOF
 ```
 
-Here, `sgrep` will search the target with the pattern `$X == $X` \(which is a stupid equals check\) and print the results to `stdout`. This also works for directories and will skip the file if parsing fails. You can specifiy the language of the pattern with `--lang javascript` for example.
+```
+$ docker run --rm -v "${PWD}:/home/repo" returntocorp/sgrep --lang python --pattern '$X == $X' test.py
+test.py
+3:if a == a:  # oops, supposed to be a == b
+```
 
-To see more options
+From here you can use our rules to search for issues in your codebase:
 
-```bash
-docker run --rm -v "${PWD}:/home/repo" returntocorp/sgrep --help
+```
+$ cd /path/to/code
+$ docker run --rm -v "${PWD}:/home/repo" returntocorp/sgrep --config https://r2c.dev/default-r2c-checks
```

Comment:
@DrewDennison why would we not use -config=r2c here?

Reply:
I'll switch to `r2c`. I wasn't aware of how the registries worked or what rules they pulled in and figured this was simplest. Just tested out `r2c` and that should be fine here :+1: